plugins/catalog-backend/src/search/index.ts

```diff
@@ -16,6 +16,7 @@
 
 export { DefaultCatalogCollatorFactory } from './DefaultCatalogCollatorFactory';
 export type { DefaultCatalogCollatorFactoryOptions } from './DefaultCatalogCollatorFactory';
+export type { CatalogCollatorEntityTransformer } from './CatalogCollatorEntityTransformer';
```

Comment:
Looks like you'll also want to export `defaultCatalogCollatorEntityTransformer` in order to allow integrators to re-use the existing transformation logic.

Reply:
done